src/main/java/com/roisoftstudio/godutch/info/paths/InfoPath.java

```diff
@@ -5,17 +5,21 @@
 
 import javax.ws.rs.GET;
 import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.Response;
 
 @Path("/")
+@Produces(MediaType.APPLICATION_JSON)
 public class InfoPath {
 
     @Inject
     private InfoService infoService;
 
     @GET
     public Response getInfo() {
-        return Response.ok(infoService.getInfo()).build();
-
+        return Response
```

Comment:
Here i would prefer a one liner since is quite simple, and for bigger builders this style what do you think?